MantleTests/MTLJSONAdapterSpec.m

```diff
@@ -591,4 +591,19 @@ + (NSValueTransformer *)NSDateJSONTransformer {
 	expect(weakTransformer).toEventually(beNil());
 });
 
+it(@"should support recursive models", ^{
+	NSDictionary *dictionary = @{
+		@"owner": @{@"name": @"Cameron"},
```

Comment:
Can you add spaces between the literal an its contents, e.g. `@{ @"name": @"Cameron" }`?